cosmo_tester/framework/handlers/softlayer.py

```diff
@@ -0,0 +1,110 @@
+########
```

Comment:
see comment about this file in the `cloudify-softlayer-plugin` repo pull request